request_information.go

```diff
@@ -465,9 +476,18 @@ func (request *RequestInformation) AddQueryParameters(source interface{}) {
 		if ok && it != nil {
 			request.QueryParameters[fieldName] = strconv.FormatInt(int64(*it), 10)
 		}
-		arr, ok := value.([]string)
+		strArr, ok := value.([]string)
+		if ok && len(strArr) > 0 {
+			request.QueryParameters[fieldName] = strings.Join(strArr, ",")

```

Comment:
```suggestion
            // populating both query parameter fields to avoid breaking compatibility with code reading this field
			request.QueryParameters[fieldName] = strings.Join(strArr, ",")

```